File: ea-domain/src/main/kotlin/com/nbe3/domain/global/util/PagingUtil.java
package com.nbe3.domain.global.util;

import com.nbe3.common.dto.Page;
import org.springframework.data.domain.PageRequest;

public class PagingUtil {

    public static PageRequest toPageRequest(Page page) {
        return PageRequest.of(page.getPage(), page.getSize());
    }
}

File: ea-domain/src/main/kotlin/com/nbe3/domain/global/util/PagingUtil.kt
package com.nbe3.domain.global.util

import com.nbe3.common.dto.Page
import org.springframework.data.domain.PageRequest

class PagingUtil {

    companion object {
        fun toPageRequest(page: Page): PageRequest {
            return PageRequest.of(page.page, page.size)
        }
    }
}
